starter-code/functions-and-arrays.js

```diff
@@ -34,7 +49,11 @@ console.log(total);
 
 // Calculate the Average
 function averageNumbers (array) {
-
+  var total = 0;
+  numbers.forEach(function(element){
+    total += element;
```

Comment:
Do you have another function that sums all the values in an array? Then, maybe you can use it here instead of repeating code.